llama-index-packs/llama-index-packs-corrective-rag/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
```

Comment:
new to contributing guide (well actually still non-existent there but will be there before EOD):

please add a section for `[tool.llamahub]` similar to this [one](https://github.com/run-llama/llama_index/blob/fc51b9fcc9c2bbdc09a9ac91deea7715872c3f44/llama-index-packs/llama-index-packs-amazon-product-extraction/pyproject.toml#L10):